src/main/java/reposense/util/FileUtil.java

```diff
@@ -40,6 +40,8 @@
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
     private static final String MESSAGE_INVALID_FILE_PATH = "\"%s\" is an invalid file path. Skipping this directory.";
+    private static final String MESSAGE_FAIL_TO_ZIP_FILES =
+            "Exception occurred while attempting to ZIP the report files.";
```

Comment:
`ZIP` -> `zip`.

Also, instead of saying `Exception occurred`, perhaps state that the report files are missing or corrupted?

Reply:
> Also, instead of saying Exception occurred, perhaps state that the report files are missing or corrupted?

I don't think we should narrow down to `report files are missing or corrupted` as many reasons can happen that it failed to zip. 

For example in this PR, the reason is due to the invalid directory being parsed in, but doesn't mean that the report files are missing or corrupted.